packages/core/src/routes/connector-data.ts

```diff
@@ -1,15 +1,6 @@
-/* eslint-disable max-lines */
 import { buildRawConnector } from '@logto/cli/lib/connector/index.js';
-import type { ConnectorFactory } from '@logto/cli/lib/connector/index.js';
-import {
-  type SmsConnector,
-  type EmailConnector,
-  demoConnectorIds,
-  VerificationCodeType,
-  validateConfig,
-} from '@logto/connector-kit';
-import { phoneRegEx, emailRegEx } from '@logto/core-kit';
-import { arbitraryObjectGuard, Connectors, ConnectorType } from '@logto/schemas';
+import { demoConnectorIds, validateConfig } from '@logto/connector-kit';
```

Comment:
Just double-check the idea behind this file change. 
- extract the /connectors/:factoryId/test API
- simply reorg the order of the rest APIs?

Reply:
Correct.